grype/presenter/sarif/presenter.go

```diff
@@ -0,0 +1,348 @@
+package sarif
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	s "github.com/owenrumney/go-sarif/sarif"
```

Comment:
`s` isn't a great import alias, why not stick with `sarif`?

Reply:
`sarif` conflicts with the package name 😬

Reply:
Oddly, I found it made things pretty quick to work with this way but if you feel strongly it should be something else, I could change it to something maybe `sarifLib` or happy to hear other suggestions. or we _could_ rename the presenter something like `sarifv1` or whatever.